.travis/docker_push.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+export CHAINERUI_LATEST_VER=`git describe --abbrev=0 --tags`
```

Comment:
I guess `$TRAVIS_TAG` can be used here?
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Reply:
thanks, I fixed to use the variable.